### ISSUE TITLE
Non-Stackable item as input

### ISSUE THREAD

**Issue by KewaiiGamer** [KewaiiGamer/RiceCraftingTable#7]
If you craft with non-stacking items you get  all the ingredienz back and the Item, that you crafted.

**Comment by GazbonicAcid** [comment]
Dupe of https://github.com/KewaiiGamer/RiceCraftingTable/issues/4
Apparently ANY recipe in Rice Crafting Table you get all of the ingrediants back.
If if that's not true, this is still a dupe of https://github.com/KewaiiGamer/RiceCraftingTable/issues/4

**Comment by KewaiiGamer** [comment]
This mod is completely broken. I won't fix it since I just can't! I wanted to port avaritia to 1.10.2 but it seems that Avaritia dev also has this bug on its mod.
Now... regarding your comment saying "ANY recipe": Apparently it's not ANY recipe. If you only use the first slot I think it works.

**Comment by ExpensiveKoala** [comment]
I fixed the issue... kinda. It has a problem I'm trying to fix. When there is only 3 or less items, it works just fine. It seems to want to add the original stacksize - 4 to the original stack. For example, 34 items turns to 64 because 34 + (34 - 4) = 64. If the stacksize is 4, it ends up doing 4 + (4 - 4) = 4.

Once I've fixed this, I'll put a pull request. Also, the issue exists for shaped and shapeless. Slot number doesn't seem to matter.

edit: realized that the reason it works with 3 or less is because 3 + (3 - 4) = 2 making it look like it worked.

**Comment by KewaiiGamer** [comment]
Thanks so much for your hard work on trying to help fixing this mod. I give you the best of the luck on fixing it. This issue has been making me give up on this mod!

**Comment by ExpensiveKoala** [comment]
I'm about to delete what you have and rewrite it because this issue is extremely weird...

**Comment by KewaiiGamer** [comment]
I wouldn't mind if you do that. When I started this mod I did it because I wanted the Dire Crafting Table for Avaritia to be available when expert mode modpacks such as FTB Infinity Evolved is available however I was still new to coding and I limited myself to copy most of the code from Avaritia and fix the errors however I didn't fix any bugs.

tl;dr you can rewrite it if you want. I won't mind!
